covizu/minimap2.py

```diff
@@ -239,10 +239,10 @@ def parse_args():
     if '.egg' in path:
         import pkg_resources
         ref_file = pkg_resources.resource_filename('covizu', 'data/NC_045512.fa')
-        vcf_file = pkg_resources.resource_filename('covizu', "data/problematic_sites_sarsCov2.vcf")
+        vcf_file = pkg_resources.resource_filename('covizu', "data/ProblematicSites_SARS-CoV2/problematic_sites_sarsCov2.vcf")
```

Comment:
If we're including these as submodules, would it be possible to load these submodules to get a path to this resource file (`.vcf`) instead of using `pkg_resources`?